
We do not want to build dietlibc twice for cross builds - just to get the
tiny wrapper for the building host ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc/diet.c	2000-01-01 00:00:00.000000000 +0100
+++ dietlibc-fixed/diet.c	2005-01-30 16:53:10.181709691 +0100
@@ -3,10 +3,13 @@
 #include <fcntl.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <write12.h>
 
 #include "dietfeatures.h"
 
+int __write2 (const char* s) {
+  return write(2, s, strlen(s));
+}
+
 /* goal:
  *   when invoked as
  * "diet gcc -c t.c"
